src/handlers/payments.ts

```diff
@@ -1,639 +1,643 @@
-import { pipe } from "fp-ts/function";
-import * as express from "express";
-import { RequestHandler } from "express";
-import * as E from "fp-ts/lib/Either";
-import * as O from "fp-ts/lib/Option";
-import * as t from "io-ts";
-import {
-  ResponseErrorForbiddenAnonymousUser,
-  ResponseErrorFromValidationErrors,
-  ResponseErrorNotFound,
-  ResponseErrorValidation,
-  ResponseSuccessJson
-} from "@pagopa/ts-commons/lib/responses";
-import { PathReporter } from "io-ts/PathReporter";
-import { PaymentResponse } from "../generated/payment_manager/PaymentResponse";
-import { sendResponseWithData, tupleWith } from "../utils/utils";
-import { TransactionResponse } from "../generated/payment_manager/TransactionResponse";
-import { IUserData } from "../constants";
-import {
-  EndpointController,
-  EndpointHandler,
-  HandlerResponseType,
-  ResponseErrorValidationFault,
-  ResponseGatewayTimeout,
-  ResponsePartyConfigurationError,
-  ResponsePaymentError,
-  ResponsePaymentStatusFaultError,
-  ResponseSuccessfulCreated,
-  ResponseUnauthorized
-} from "../utils/types";
-import {
-  ActivatePaymentT,
-  GetActivationStatusT,
-  GetPaymentInfoT
-} from "../generated/pagopa_proxy/requestTypes";
-import { RptId } from "../generated/pagopa_proxy/RptId";
-import { PaymentRequestsGetResponse } from "../generated/pagopa_proxy/PaymentRequestsGetResponse";
-import { PaymentFaultEnum } from "../generated/pagopa_proxy/PaymentFault";
-import { PaymentActivationsPostRequest } from "../generated/pagopa_proxy/PaymentActivationsPostRequest";
-import {
-  FlowCase,
-  getFlowCookie,
-  getFlowFromRptId,
-  maybeGetFlowCookie,
-  setFlowCookie
-} from "../flow";
-import { PaymentActivationsPostResponse } from "../generated/pagopa_proxy/PaymentActivationsPostResponse";
-import { PaymentActivationsGetResponse } from "../generated/pagopa_proxy/PaymentActivationsGetResponse";
-import { Pay3ds2UsingPOSTT } from "../generated/payment_manager/requestTypes";
-import { logger } from "../logger";
-import { PayRequest } from "../generated/payment_manager/PayRequest";
-import { GatewayFaultEnum } from "../generated/pagopa_proxy/GatewayFault";
-import { PartyConfigurationFaultEnum } from "../generated/pagopa_proxy/PartyConfigurationFault";
-import { PartyTimeoutFaultEnum } from "../generated/pagopa_proxy/PartyTimeoutFault";
-import { PaymentStatusFaultEnum } from "../generated/pagopa_proxy/PaymentStatusFault";
-import { ValidationFaultEnum } from "../generated/pagopa_proxy/ValidationFault";
-import { ActivationState, store } from "../store/store";
+/**
```

Comment:
why not delete this file completely? can always be restored from git history, if needed

Reply:
I can't do it, _"troppi ricordi"_

Reply:
![image](https://github.com/pagopa/pagopa-checkout-be-mock/assets/115724836/72b450cc-32ea-4284-a65b-acb809ea9b83)